### ISSUE TITLE
suggestion: npm outdated should include package type

### ISSUE THREAD

**Issue by pgilad** [npm/npm#6759]
currently running `npm outdated` outputs: 
```sh
❯ npm outdated --depth=0
Package            Current  Wanted  Latest  Location
gulp-autoprefixer    1.0.1   1.0.1   2.0.0  gulp-autoprefixer
gulp-concat          2.4.1   2.4.2   2.4.2  gulp-concat
coffeeify            0.7.0   0.7.0   1.0.0  coffeeify
gulp-sass            1.2.3   1.2.3   1.1.0  gulp-sass
```

It does not output the package type (`devDependency`, `dependency`, `peerDependency`, etc..)
In a project with many modules, that could be useful information, especially for knowing when your production dependencies are outdated. I constantly find myself doing `cat package.json` to see if a certain dependency is in my `dependencies` or `devDependencies`.

So my suggestion is to add another table column, listing `Package Type`.

So in our above example:
```bash
❯ npm outdated --depth=0
Package            Current  Wanted  Latest  Location           Package Type
gulp-autoprefixer    1.0.1   1.0.1   2.0.0  gulp-autoprefixer  devDependency
gulp-concat          2.4.1   2.4.2   2.4.2  gulp-concat        devDependency
coffeeify            0.7.0   0.7.0   1.0.0  coffeeify          devDependency
gulp-sass            1.2.3   1.2.3   1.1.0  gulp-sass          devDependency
```

**Comment by othiym23** [comment]
This is included in `npm@2.5.0` (currently `npm@next`), thanks to @watilde. `npm outdated --depth=0 --long` will give output pretty much identical to what you include above. :tada:

**Comment by pgilad** [comment]
:clap: Yey!!!
